services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -655,35 +722,34 @@ public void processDlqRequestRejected() {
         individualRequest.setErrorLog(errorLog);
         individualRequest.setRequestStatus(IIndividualRequest.IndividualRequestStatus.REJECTED.getStatus());
 
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
-
         final IBulkSubmission bulkSubmission = individualRequest.getBulkSubmission();
 
-        EasyMock.expect(
-                this.mockIndividualRequestDao.queryAsCount(EasyMock.same(IndividualRequest.class), EasyMock.isA(Supplier.class))).andReturn(0L);
+        when(
+                this.mockIndividualRequestDao.queryAsCount(eq(IndividualRequest.class), isA(Supplier.class))).thenReturn(0L);
 
         mockIndividualRequestDao.persist(bulkSubmission);
-        EasyMock.expectLastCall();
+        verify(mockIndividualRequestDao).persist(bulkSubmission);
 
-        EasyMock.replay(mockIndividualRequestDao);
-        EasyMock.replay(mockCacheable);
-        EasyMock.replay(mockErrorMsgCacheable);
+        ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
+        listAppender.start();
+        logger.addAppender(listAppender);
 
         this.targetAppSubmissionService.processDLQRequest(individualRequest, requestStatus);
 
-        EasyMock.verify(mockIndividualRequestDao);
-        EasyMock.verify(mockCacheable);
-        EasyMock.verify(mockErrorMsgCacheable);
+        List<ILoggingEvent> logList = listAppender.list;
 
-        Assert.assertEquals("Bulk submission status is incorrect", IBulkSubmission.BulkRequestStatus.COMPLETED
-                .getStatus(), individualRequest.getBulkSubmission().getSubmissionStatus());
-        Assert.assertNotNull("Bulk submission completed date should be populated", individualRequest
-                .getBulkSubmission().getCompletedDate());
-        Assert.assertNotNull("Bulk submission updated date should be populated", individualRequest
-                .getBulkSubmission().getUpdatedDate());
-        Assert.assertEquals("Individual Request should not be marked as dead letter", false,
-                individualRequest.isDeadLetter());
+        assertFalse(logList.isEmpty());
+
+        logger.detachAndStopAllAppenders();
+
+        assertEquals( IBulkSubmission.BulkRequestStatus.COMPLETED.getStatus(),
+                      individualRequest.getBulkSubmission().getSubmissionStatus(),"Bulk submission status is incorrect");
+        assertNotNull(individualRequest
+                .getBulkSubmission().getCompletedDate(), "Bulk submission completed date should be populated");
+        assertNotNull(individualRequest
+                .getBulkSubmission().getUpdatedDate(),"Bulk submission updated date should be populated");
+        assertEquals( false,
```

Comment:
Could use assertFalse() here instead of assertEquals().

Reply:
done